src/main/java/net/neoforged/neoforge/items/wrapper/SidedInvWrapper.java

```diff
@@ -47,6 +47,9 @@ public SidedInvWrapper(WorldlyContainer inv, @Nullable Direction side) {
     }
 
     public static int getSlot(WorldlyContainer inv, int slot, @Nullable Direction side) {
```

Comment:
This method should be made protected, the `@Nullable` removed, and the usages updated accordingly. `WorldlyContainer` doesn't have a concept of slots for a null side.

Reply:
This cannot be done on older versions as it would be a breaking change. I am retargeting this PR for 1.21.1 and will open a new one for latest with that change in mind.

Reply:
Never mind, I forgot that will attempt to bring in the rest of the branch.

Reply:
There is no point in doing this, it will just make the call sites of `getSlot` more annoying because the `null` side will need to be handled there explicitly.